galaxy_ng/app/tasks/publishing.py

```diff
@@ -3,15 +3,21 @@
 from django.conf import settings
 from django.contrib.contenttypes.models import ContentType
 from django.utils.translation import gettext_lazy as _
-
-from pulpcore.plugin.models import Task
 from pulp_ansible.app.models import AnsibleDistribution, AnsibleRepository, CollectionVersion
 from pulp_ansible.app.tasks.collections import import_collection
+from pulpcore.plugin.models import Task
 
 from .promotion import call_copy_task, call_remove_task
 
 log = logging.getLogger(__name__)
 
+api_access_log = logging.getLogger("automated_logging")
+formatter = logging.Formatter("%(asctime)s %(levelname)s %(message)s")
+handler = logging.FileHandler("/var/log/galaxy_api_access.log")
+handler.setFormatter(formatter)
+api_access_log.setLevel(logging.INFO)
+api_access_log.addHandler(handler)
```

Comment:
i assume this logger is accessible to django throughout app, can we use it? https://github.com/ansible/galaxy_ng/blob/c2eb855c2229dc20adf894e8d7887006b14f966a/galaxy_ng/app/dynaconf_hooks.py#L180-L185

Reply:
Ha, nothing but errors yesterday when trying to reuse the global logger.  Today, worked immediately.  :man_shrugging: